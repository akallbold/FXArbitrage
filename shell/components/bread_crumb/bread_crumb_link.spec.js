import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import BreadCrumbLink from './bread_crumb_link'

describe('<BreadCrumbLink />', () => {
  it('renders', () => {
    const breadCrumbLink = <BreadCrumbLink
      brandId='5'
      channel='snapchat'
      childrenLevel='adsets'
      name='My Cool Selected Campaign'
    />
    const wrapper = shallow(breadCrumbLink)

    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
