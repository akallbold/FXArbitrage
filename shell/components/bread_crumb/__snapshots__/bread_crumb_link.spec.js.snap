// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BreadCrumbLink /> renders 1`] = `
<Connect(PersistentQueryLink)
  className="breadcrumb"
  href={
    Object {
      "pathname": "/brands/5/snapchat/manager/adsets",
      "persistQuery": true,
    }
  }
  title="My Cool Selected Campaign"
>
  My Cool Selected Campaign
</Connect(PersistentQueryLink)>
`;
